src/App.test.jsx

```diff
@@ -1,9 +1,5 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import React from 'react';
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
```

Comment:
Is this sticking around for something later or can it be cleaned up?

Reply:
I have no idea.... I'll look into it now

Reply:
So at the moment it is pulling the test library from react, I can add just the basic test for it to ensure rendering the app doesn't crash it but I'd have to think about more advanced testing (since I know none) @DewaldBez @auretswanepoel 

`import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("renders App component", () => {
render(<App />);
});`